src/api/public/apidocs-new/components/parameters/request_withhistory.yaml

```diff
@@ -0,0 +1,9 @@
+in: query
+name: withhistory
+schema:
+  type: string
+description: |
+  Add the request history.
```

Comment:
Same here. What about "Include" or "Display" instead of "Add"?

Reply:
Also changed to "Include".